server/src/services/cars.ts

```diff
@@ -120,6 +120,11 @@ const getCarByCustomerId = async (customerId) => {
   });
   return car;
 };
+
+const addImageService = async ({ image, carId }) => {
+  const rows = await Image.create({ ...{ image, carId } });
```

Comment:
```suggestion
const rows = await Image.create({ image, carId });
```
why use ... you can put the image and carId without  ...